packages/public-api/src/index.ts

```diff
@@ -14,7 +14,6 @@ checkRequiredEnvironmentVariables([
   Env.SqsWorkerQueueUrl,
   Env.InternalApiRootUrl,
   Env.GradeJsApiKey,
-  Env.RollbarApiKey,
```

Comment:
Why have you removed it? You can provide any random string as a token for the dev environment

Reply:
like 
```
export ROLLBAR_API_TOKEN=test
```